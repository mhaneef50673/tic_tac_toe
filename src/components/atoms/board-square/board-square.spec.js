import React from "react";
import { mount } from "enzyme";
import { mountToJson } from "enzyme-to-json";
import BoardSquare from "./index";

describe("BoardSquare Component", () => {
  it("should render the component", () => {
    const tree = mount(
      <BoardSquare
        isWinning={true}
        squareValue={1}
        squareClickHanlder={() => {}}
      />
    );
    expect(mountToJson(tree)).toMatchSnapshot();
  });
});
