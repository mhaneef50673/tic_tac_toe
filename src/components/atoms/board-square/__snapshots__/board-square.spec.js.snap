// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BoardSquare Component should render the component 1`] = `
<BoardSquare
  isWinning={true}
  squareClickHanlder={[Function]}
  squareValue={1}
>
  <button
    className="board-square winning-square"
    onClick={[Function]}
  >
    1
  </button>
</BoardSquare>
`;
